sim/hunter/runes.go

```diff
@@ -248,10 +249,10 @@ func (hunter *Hunter) applyLockAndLoad() {
 
 	lockAndLoadMetrics := hunter.Metrics.NewResourceMetrics(core.ActionID{SpellID: 415413}, proto.ResourceType_ResourceTypeMana)
 
-	icd := core.Cooldown{
-		Timer:    hunter.NewTimer(),
-		Duration: time.Second * 8,
-	}
+	// icd := core.Cooldown{
+	// 	Timer:    hunter.NewTimer(),
+	// 	Duration: time.Second * 8,
+	// }
```

Comment:
Didn't the patch notes says lock and load now has an 8s ICD?

Reply:
Icd was removed as of the latest ptr build